iDiff/differs/differs.go

```diff
@@ -1,16 +1,65 @@
 package differs
 
-import "errors"
+import (
+	"errors"
+	"os"
+
+	"github.com/runtimes-common/iDiff/utils"
+)
 
 var diffs = map[string]func(string, string) (string, error){
 	"hist": History,
 	"dir":  Package,
+	"apt":  AptDiff,
 }
 
 func Diff(arg1, arg2, differ string) (string, error) {
 	if f, exists := diffs[differ]; exists {
-		return f(arg1, arg2)
+		if differ == "hist" {
```

Comment:
I think all differs are going to require the ImageToDir work (provided they are passed images and not directories) because without it there is no file directory to inspect.  Can definitely revisit this as necessary though.